src/components/new-safe-apps/SafeAppPreviewDrawer/SafeAppPreviewDrawer.tsx

```diff
@@ -4,32 +4,85 @@ import Drawer from '@mui/material/Drawer'
 import Box from '@mui/material/Box'
 import Typography from '@mui/material/Typography'
 import Button from '@mui/material/Button'
+import { IconButton, SvgIcon } from '@mui/material'
+import Tooltip from '@mui/material/Tooltip'
 import type { SafeAppData } from '@safe-global/safe-gateway-typescript-sdk'
 
 import { getSafeAppUrl } from '@/components/new-safe-apps/SafeAppCard/SafeAppCard'
+import ChainIndicator from '@/components/common/ChainIndicator'
+import SafeAppIconCard from '../SafeAppIconCard/SafeAppIconCard'
+import SafeAppActionButtons from '../SafeAppActionButtons/SafeAppActionButtons'
+import CloseIcon from '@/public/images/common/close.svg'
```

Comment:
Maybe we can use the [Material UI Close icon](https://mui.com/material-ui/material-icons/?query=close&selected=Close)?

Reply:
I did it but as the other 2 icons are custom ones the close from material feels different and more padded so i need to play with viewBox or other styles to fit it

![image](https://user-images.githubusercontent.com/1576776/212278628-efb67db8-48ee-4265-a9ad-a5a8470e02da.png)

Then on hover the border-radius as well need to be moved 

![image](https://user-images.githubusercontent.com/1576776/212279326-35c26b20-d7f9-4314-b72e-041ae8b81b5f.png)

... so finally decided to add the custom one as the others. If you prefer the material one i can try to fit it